src/helpers.js

```diff
@@ -205,7 +205,7 @@ export function normalizeSwagger(parsedSpec) {
                 operation[inheritName] = inherits[inheritName];
               } else if (inheritName === 'parameters') {
                 // eslint-disable-next-line no-restricted-syntax
-                for (const param of inherits[inheritName]) {
+                for (const param of inherits[inheritName].slice(0).reverse()) {
```

Comment:
Why not use a vanilla for loop and iterate in reverse order , that would save one extra loop that happens with **.reverse**  and  **.slice()**